config.json

```diff
@@ -2315,239 +2323,255 @@
         "groupId": "androidx.activity",
         "artifactId": "activity",
         "version": "1.7.2",
-        "nugetVersion": "1.7.2",
+        "nugetVersion": "1.7.2.1",
         "nugetId": "Xamarin.AndroidX.Activity",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.annotation",
         "artifactId": "annotation",
         "version": "1.6.0",
-        "nugetVersion": "1.6.0.3",
+        "nugetVersion": "1.6.0.4",
         "nugetId": "Xamarin.AndroidX.Annotation",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.appcompat",
         "artifactId": "appcompat",
         "version": "1.6.1",
-        "nugetVersion": "1.6.1.2",
+        "nugetVersion": "1.6.1.3",
         "nugetId": "Xamarin.AndroidX.AppCompat",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.browser",
         "artifactId": "browser",
-        "version": "1.5.0",
-        "nugetVersion": "1.5.0.2",
+        "version": "1.6.0",
+        "nugetVersion": "1.6.0",
         "nugetId": "Xamarin.AndroidX.Browser",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.cardview",
         "artifactId": "cardview",
         "version": "1.0.0",
-        "nugetVersion": "1.0.0.20",
+        "nugetVersion": "1.0.0.21",
         "nugetId": "Xamarin.AndroidX.CardView",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.collection",
         "artifactId": "collection",
         "version": "1.2.0",
-        "nugetVersion": "1.2.0.8",
+        "nugetVersion": "1.2.0.9",
         "nugetId": "Xamarin.AndroidX.Collection",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.concurrent",
         "artifactId": "concurrent-futures",
         "version": "1.1.0",
-        "nugetVersion": "1.1.0.13",
+        "nugetVersion": "1.1.0.14",
         "nugetId": "Xamarin.AndroidX.Concurrent.Futures",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.constraintlayout",
         "artifactId": "constraintlayout",
         "version": "2.1.4",
-        "nugetVersion": "2.1.4.5",
+        "nugetVersion": "2.1.4.6",
         "nugetId": "Xamarin.AndroidX.ConstraintLayout",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.core",
         "artifactId": "core",
         "version": "1.10.0",
-        "nugetVersion": "1.10.1.1",
+        "nugetVersion": "1.10.1.2",
         "nugetId": "Xamarin.AndroidX.Core",
         "dependencyOnly": true
       },
+      {
+        "groupId": "androidx.datastore",
+        "artifactId": "datastore",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
+        "nugetId": "Xamarin.AndroidX.Datastore",
```

Comment:
I guess NuGet package id's aren't case-sensitive, since this appears to work, but it should be capitalized `Xamarin.AndroidX.DataStore`.

Reply:
Right. NuGet is case insensitive.
Fixed